manifest.json

```diff
@@ -33,31 +36,34 @@
         "scripts/angular-route.min.js",
         "scripts/angular-animate.min.js",
         "scripts/bootstrap-tpls.min.js",
-        "pages/init.js",
-        "pages/directives.js",
-        "pages/services.js",
-        "pages/baseController.js",
-        "pages/allDatacontroller.js",
-        "pages/routeConfig.js"
+        "js/init.js",
+        "js/directives.js",
+        "js/services.js",
+        "js/baseController.js",
+        "js/allDataController.js",
```

Comment:
Realized the case difference when tried to unpack the extension in Linux. The Linux file system is case sensitive and windows isn't. That is why it was working properly in Windows.